python/lsst/ts/FiberSpectrograph/csc.py

```diff
@@ -51,6 +55,11 @@ class FiberSpectrographCsc(salobj.BaseCsc):
     outpath : `str`, optional
         Write output files to this path.
         TODO: this is temporary until we have a working LFA.
+    index : `int`, optional
+        The SAL index; this determines which spectrograph to connect to.
+        See the ``FiberSpectrograph`` Enumeration in
+        ``ts_xml/sal_interfaces/SALSubsystems.xml`` for index:name mapping.
+        index=0 means use the only connected spectrograph.
```

Comment:
0 is not a safe SAL index for an indexed SAL component because it will see DDS messages sent to any of the FiberSpectrograph instances. Please use some other value. I agree that the values associated with a particular spectrograph should match the enumeration in ts_xml. I suppose that leaves 4? -1?

Reply:
I will use `-1` to make it obviously distinct from the explicit device indexes.